content/docs/command-reference/commit.md

```diff
@@ -280,4 +280,4 @@ Data and pipelines are up to date.
 ```
 
 Instead of reproducing the pipeline for changes that do not produce different
-results, just use `commit` on both Git and DVC.
+results, use `commit` on both Git and DVC.
```

Comment:
Again, it means "only" here, as a selling point. (As opposed to having to repro the whole pipeline). Please revert this one.